tests/WE2E/test_configs/grids_extrn_mdls_suites_community/config.grid_RRFS_CONUScompact_25km_ics_RRFS_lbcs_RRFS_suite_RRFS_v1beta.yaml

```diff
@@ -0,0 +1,35 @@
+metadata:
+  description: |-
+    This test is to ensure that the workflow running in community mode
+    completes successfully on the RRFS_CONUScompact_25km grid using the RRFS_v1beta
+    physics suite with ICs and LBCs derived from the RRFS forecasts mapped onto
+    3-km regular grid (rrfs*.conus.grib2). This test uses old v1 sfc_data,
+    not the v2 fractional grid sfc_data.
+user:
+  RUN_ENVIR: community
+workflow:
+  CCPP_PHYS_SUITE: FV3_RRFS_v1beta
+  PREDEF_GRID_NAME: RRFS_CONUScompact_25km
+  DATE_FIRST_CYCL: '2024060517'
+  DATE_LAST_CYCL: '2024060517'
+  FCST_LEN_HRS: 9
+  PREEXISTING_DIR_METHOD: rename
+task_make_orog:
+  OMP_NUM_THREADS_MAKE_OROG: 24
```

Comment:
This is a huge number of threads comparatively. Can you please provide justification?

Reply:
This could be needed when full-grid RRFS data is processed, and 3-km grid is used. It may not be needed for 25-km CONUS grid test.

Reply:
It should be reduced to what is needed for this test case.

Reply:
done